src/to-fresh.js

```diff
@@ -71,6 +71,7 @@ Convert JRS resume sections to FRESH.
             start: edu.startDate,
             end: edu.endDate,
             grade: edu.gpa,
+            title: edu.title,
```

Comment:
There's definitely no direct equivalent. Here's the description of `title` in the FRESH schema: 

> A freeform title for this education stint. Typically, this should be the short name of your degree, certification, or training.

A good solution might be combining json resume's `area` and `studyType` fields. Here are their definitions:

         "area": {
            "type": "string",
            "description": "e.g. Arts"
          },
          "studyType": {
            "type": "string",
            "description": "e.g. Bachelor"
          }

So I'm thinking a good substitute might be `studyType + ' of ' + area`. If not that, then I think leaving it out is fine. What are your thoughts?